src/views/blocks/parts/details.php

```diff
@@ -131,7 +142,7 @@
 		// Event Cost
 		if ( ! empty( $cost ) ) : ?>
 
-            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ) ?> </dt>
+            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ); ?> </dt>
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 3d4196da3416e259e93f24c1c24c3f87</sub>